Modules/Components/Niftyreg/include/selxNiftyregf3dComponent.hxx

```diff
@@ -106,6 +115,26 @@ Niftyregf3dComponent<  TPixel >
 
 }
 
+template< class TPixel >
+bool
+Niftyregf3dComponent< TPixel >
+::ConnectionsSatisfied()
```

Comment:
We could perhaps also add debug-level logging statements in the default implementation (not just this function, but others as well) so users have runtime debug information during development. The overhead is almost zero especially when it happens only during instantiation.

Reply:
Yes, good idea. Made it issue #118.